docs/source/workflow_inputs.rst

```diff
@@ -1554,6 +1554,15 @@ Additional information is entered in each ``Ducts``.
          See the `Building America Solution Center <https://basc.pnnl.gov/resource-guides/ducts-buried-attic-insulation>`_ for more information.
   .. [#] DuctLocation choices are "conditioned space", "basement - conditioned", "basement - unconditioned", "crawlspace - unvented", "crawlspace - vented", "attic - unvented", "attic - vented", "garage", "outside", "exterior wall", "under slab", "roof deck", "other housing unit", "other heated space", "other multifamily buffer space", or "other non-freezing space".
          See :ref:`hpxmllocations` for descriptions.
+  .. [#] The sum of all FractionDuctArea must each equal to 1, both for the supply side and return side.
+  .. [#] If both are provided, DuctSurfaceArea will be used in the model.
+  .. [#] Duct surface areas will be calculated based on BSR/RESNET/ICC 301-2022 Table 4.2.2(1) footnote z. aa.:
```

Comment:
Rather than pointing to the draft document, let's reference the (pending) ANSI standard instead. And since table numbers and footnotes could change, let's just leave that out.

```suggestion
  .. [#] Duct surface areas will be calculated based on ANSI/RESNET/ICC 301-2022:
```